enterprise_subsidy/apps/content_metadata/constants.py

```diff
@@ -0,0 +1,7 @@
+"""
+Constants about content metadata.
+"""
+EDX_PRODUCT_SOURCE = "edX"
+TWOU_PRODUCT_SOURCE = "2u"
+EXECUTIVE_EDUCATION_MODE = "paid-executive-education"
+EDX_VERIFIED_COURSE_MODE = "verified"
```

Comment:
Nit: should enum options be [grouped into classes](https://docs.python.org/3/howto/enum.html#using-a-descriptive-string)? something like:

```suggestion
from enum import Enum

class ProductSource(Enum)
    """
    TODO
    """
    EDX = "edX"  # e.g. OCM courses
    TWOU = "2u"  # e.g. ExecEd courses

etc.
```